pkg/gce-cloud-provider/compute/cloud-disk.go

```diff
@@ -134,8 +115,6 @@ func (d *CloudDisk) GetPDType() string {
 		pdType = d.disk.Type
 	case d.betaDisk != nil:
 		pdType = d.betaDisk.Type
-	case d.alphaDisk != nil:
```

Comment:
How will this impact the endpoint in TPC?

Reply:
I think just that in TPC it will also only use beta/v1 which should not be a problem